src/payment/PaymentPage.test.jsx

```diff
@@ -35,36 +41,61 @@ jest.mock('universal-cookie', () => {
 // Mock language cookie
 Object.defineProperty(global.document, 'cookie', {
   writable: true,
-  value: `${App.config.LANGUAGE_PREFERENCE_COOKIE_NAME}=en`,
+  value: `${getConfig().LANGUAGE_PREFERENCE_COOKIE_NAME}=en`,
 });
-App.apiClient = jest.fn();
 
-jest.mock('@edx/frontend-analytics', () => ({
+jest.mock('@edx/frontend-platform/analytics', () => ({
   sendTrackEvent: jest.fn(),
 }));
 
+configureI18n({
+  config: {
+    ENVIRONMENT: process.env.ENVIRONMENT,
+    LANGUAGE_PREFERENCE_COOKIE_NAME: process.env.LANGUAGE_PREFERENCE_COOKIE_NAME,
+  },
+  loggingService: {
+    logError: jest.fn(),
+    logInfo: jest.fn(),
+  },
+  messages: {
+    uk: {},
+    th: {},
+    ru: {},
+    'pt-br': {},
+    pl: {},
+    'ko-kr': {},
+    id: {},
+    he: {},
+    ca: {},
+    'zh-cn': {},
+    fr: {},
+    'es-419': {},
+    ar: {}
+  },
+});
+
+const authenticatedUser = Factory.build('userAccount');
+
 describe('<PaymentPage />', () => {
   let store;
 
   beforeEach(() => {
-    const userAccount = Factory.build('userAccount');
-
     store = createStore(createRootReducer(), {}, applyMiddleware(thunkMiddleware));
-    store.dispatch(fetchUserAccountSuccess(userAccount));
   });
 
   describe('Renders correctly in various states', () => {
     beforeEach(() => {
       analytics.sendTrackingLogEvent = jest.fn();
-      Cookies.result = undefined;
```

Comment:
Deleting this seems wrong - it means the cookie will exist for these tests, suddenly, when their default state was apparently supposed to be one where they had no currency/language cookies (undefined)

Reply:
Added back in in a form – the cookie mock mechanism was updated some.